src/closing-disclosure/index.html

```diff
@@ -41,22 +41,36 @@ <h1 class="page-title">Closing Disclosure Explainer</h1>
         {% set process_links = [
             { 
               'desc': 'Already chosen a loan?',
-              'label': 'Get ready to close', 
-              'url': '/owning-a-home/process/close/'
+              'urls': [
```

Comment:
Can you add an issue to the backlog on github about these Related Links? Right now you have to edit every page that has them and this seems like something that should be fixed.